### PR TITLE
8294691: dynamicArchive/RelativePath.java is running other test case

### PR DESCRIPTION
**Pull request by matias9927**
Test case in RelativePath.java corrected. Verified with tier 1-4 tests.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8294691](https://bugs.openjdk.org/browse/JDK-8294691): dynamicArchive/RelativePath.java is running other test case

### Reviewers
 * [Ioi Lam](https://openjdk.org/census#iklam) (@iklam - **Reviewer**) ⚠️ Review applies to [3a92d84d](https://git.openjdk.org/jdk/pull/10597/files/3a92d84d80bc3bfdaa9d8ead131b5cf849f6213b)
 * [Calvin Cheung](https://openjdk.org/census#ccheung) (@calvinccheung - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/10597/head:pull/10597` \
`$ git checkout pull/10597`

Update a local copy of the PR: \
`$ git checkout pull/10597` \
`$ git pull https://git.openjdk.org/jdk pull/10597/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10597`

View PR using the GUI difftool: \
`$ git pr show -t 10597`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/10597.diff">https://git.openjdk.org/jdk/pull/10597.diff</a>

</details>

### CODE REVIEW

File: test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/RelativePath.java [hunk 1]
```diff
@@ -54,6 +54,16 @@ private static void doTest(String topArchiveName) throws Exception {
...
+
+        // Create CDS Archive
+        dump(topArchiveName, "-Xlog:cds",
+            "-Xlog:cds+dynamic=debug",
+            "-cp", "." + File.separator + "hello.jar" + File.pathSeparator + appJar2,
+            "HelloMore")
+            .assertNormalExit(output-> {
+                    output.shouldContain("Written dynamic archive 0x");
+            });
+
```
Comment (iklam) [inline test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/RelativePath.java:66]: The new block also uses a relative path so it should be moved to under the `// relative path starting with "."` line.

**Review by iklam — APPROVED** [review]
Looks good, just a small nit.

File: test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/RelativePath.java [hunk 2]
```diff
@@ -54,6 +54,16 @@ private static void doTest(String topArchiveName) throws Exception {
         int idx = appJar.lastIndexOf(File.separator);
         String jarName = appJar.substring(idx + 1);
         String jarDir = appJar.substring(0, idx);
+
+        // Create CDS Archive
+        dump(topArchiveName, "-Xlog:cds",
+            "-Xlog:cds+dynamic=debug",
+            "-cp", "." + File.separator + "hello.jar" + File.pathSeparator + appJar2,
```
Comment (calvinccheung) [inline test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/RelativePath.java:61]: For dumping archive, I think it's better not to use relative path as follows:
`"-cp", appJar + File.pathSeparator + appJar2,`

This test is for validating using relative path during runtime works while relative path wasn't used during dump time.

**Review by calvinccheung — COMMENTED** [review]
Just one comment below.

**Review by calvinccheung — APPROVED** [review]
Update looks good. Thanks.

**Pull request closed without merging**